cves/CVE-2013-0894.yml

```diff
@@ -361,8 +404,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: 119
+CWE_note: |
+  CWE 119 and 120 were found. 119 was involved with improper Restriction of
```

Comment:
What is improper restriction of operation? Can you elaborate on that?

Reply:
it was stated in the next sentence